de-lib/src/main/java/org/iplantc/de/apps/client/events/tools/DeleteToolSelected.java

```diff
@@ -0,0 +1,46 @@
+package org.iplantc.de.apps.client.events.tools;
+
+import org.iplantc.de.client.models.tool.Tool;
+
+import com.google.common.base.Preconditions;
+import com.google.gwt.event.shared.EventHandler;
+import com.google.gwt.event.shared.GwtEvent;
+import com.google.gwt.event.shared.HandlerRegistration;
+
+import java.util.List;
+
+/**
+ * Created by jstroot on 3/5/15.
```

Comment:
Ya I copied most of events for an App and changed it for Tools :). I will get that updated.